### ISSUE TITLE
Missing activation functions under "Available activations" section in keras-core docs

### ISSUE THREAD

**Issue by cloudgiant** [keras-team/keras-io#1537]
While looking for `gelu` activation function, I have discovered that the [Available activations](https://keras.io/keras_core/api/layers/activations/#available-activations) section which is autogenerated doesn't contain all available activation functions under `keras_core.activations`. These are:

- leaky_relu
- relu6
- silu
- gelu
- hard_sigmoid
- linear
- mish
- log_softmax

Is there a reason why they are excluded? If not, the list defined in 
https://github.com/keras-team/keras-io/blob/7d5365d5bc4e66585ec620991a4eee17e51e0595/scripts/keras_core_api_master.py#L121-L130

should be adjusted to include these missing activation functions for completeness' sake.

**Comment by SuryanarayanaY** [comment]
Hi @cloudgiant ,

Thanks for reporting. I acknowledge few activations like `gelu,hard-sigmoid,linear & mish` missing in the keras documentation but available in tf.keras documentation.These activations available in keras-core activations.py file which means may be we can add documentation there.

We will review internally and update.

Thankyou!

**Comment by qlzh727** [comment]
Thanks for reporting the issue, would u mind send a PR to fix this?
